apps/mobile/src/libs/components/components.tsx

```diff
@@ -4,6 +4,7 @@ export { Button } from "./button/button";
 export { Input } from "./input/input";
 export { Link } from "./link/link";
 export { LoaderWrapper } from "./loader-wrapper/loader-wrapper";
+export { Notification } from "./notification/notification";
```

Comment:
As soon as we'll have a separate service for in-app notifications, let's rename this to `ToastMessage` and use like `tastMessage.info(....)`

Reply:
Do I need to rename component Notification or  notification.package.ts and notification.ts to use it as toastMessage.info?

Reply:
Yes, please rename it to ToastMessage as well